components/splitter/SplitBar.tsx

```diff
@@ -42,12 +45,15 @@ const SplitBar: React.FC<SplitBarProps> = (props) => {
     onOffsetUpdate,
     onOffsetEnd,
     onCollapse,
+    lazy,
+    containerSize,
   } = props;
 
   const splitBarPrefixCls = `${prefixCls}-bar`;
 
   // ======================== Resize ========================
   const [startPos, setStartPos] = useState<[x: number, y: number] | null>(null);
+  const [constrainedOffset, setConstrainedOffset] = useState<[x: number, y: number] | null>(null);
```

Comment:
```suggestion
  const [constrainedOffset, setConstrainedOffset] = useState<number | null>(null);
```

constrainedOffset 是一个偏移数字就够了，不需要是数组。css 变量也是一样，不需要给两个，通过 vertical 判断后给具体的 translate。